admin/app/assets/javascripts/workarea/admin/modules/active_by_segment_tooltips.js

```diff
@@ -58,36 +58,47 @@ WORKAREA.registerModule('activeBySegmentTooltips', (function () {
                     .remove();
         },
 
-        getConfig = function (trigger) {
-            var $link = $(trigger),
-                $form = $link.closest('form'),
-                $content = $(trigger.hash);
+        updateFields = function ($content, $trigger) {
+            var $form = $trigger.closest('form');
 
+            removeHiddenInputs($form);
+            addHiddenInputs($form, $content);
+            updateMessages($trigger, $content);
+        },
+
+        getConfig = function (trigger, $content) {
             return _.assign({}, WORKAREA.config.tooltipster, {
-                trigger: 'click',
                 interactive: true,
                 content: $content,
-                functionAfter: function (instance) {
-                    var $tooltipContent = instance.content();
-                    removeHiddenInputs($form);
-                    addHiddenInputs($form, $tooltipContent);
-                    updateMessages($link, $tooltipContent);
+                trigger: 'custom',
+                triggerOpen: {
+                    click: true
+                },
+                triggerClose: {
+                    mouseleave: true
                 }
             });
         },
 
+        setup = function (index, trigger) {
+            var $content = $(trigger.hash),
+                $select = $content.find('[data-select]'),
+                handleChange = _.partial(updateFields, $content, $(trigger));
```

Comment:
why is `$(trigger)` being passed into this function, but you don't seem to use it?